content/blog/2018/egress-monitoring-access-control.md

```diff
@@ -0,0 +1,546 @@
+weight: 86
+keywords: [egress, access-control, monitoring]
+---
+
+While the main focus of Istio is managing traffic between the microservices inside a service mesh, Istio can also  manage ingress (from outside into the mesh) and egress (from the mesh outwards) traffic. Istio can uniformly enforce access policies and aggregate telemetry data for mesh-internal, ingress and egress traffic.
+
+In this blog post we show how Istio monitoring and access policies are applied to HTTP egress traffic. The instructions in this blog post are valid for Istio [0.8.0](https://github.com/istio/istio/releases/tag/0.8.0) or later.
+
+## Use case
+
+Consider an organization that runs applications that process content of _cnn.com_. The applications are decomposed into microservices deployed in an Istio service mesh. The applications access pages of various topics of _cnn.com_: [edition.cnn.com/politics](https://edition.cnn.com/politics), [edition.cnn.com/sport](https://edition.cnn.com/sport) and  [edition.cnn.com/health](https://edition.cnn.com/health). The organization [configures Istio to allow access to edition.cnn.com](/docs/tasks/traffic-management/egress-tls-origination/) and everything works fine. However, at some point in time the organization decides to banish politics. Practically, it means blocking access to [edition.cnn.com/politics](https://edition.cnn.com/politics) and allowing access to [edition.cnn.com/sport](https://edition.cnn.com/sport) and  [edition.cnn.com/health](https://edition.cnn.com/health) only. The organization will grant permissions to individual applications, to applications in particular namespaces and to particular users to access [edition.cnn.com/politics](https://edition.cnn.com/politics), on a case-by-case basis.
+
+To achieve that goal, the organization's operations people will monitor the access to the external services and will analyze the Istio logs to verify that no unauthorized request was sent to [edition.cnn.com/politics](https://edition.cnn.com/politics). They will also configure Istio to prevent access to [edition.cnn.com/politics](https://edition.cnn.com/politics) automatically.
+
+The organization is resolved to prevent any tampering with the new policy. It decided to put mechanisms in place that will prevent any possibility for a malicious application to access the forbidden topic.
+
+## Related tasks
+
+The [Control Egress Traffic](/docs/tasks/traffic-management/egress/) task demonstrates how external (outside the Kubernetes cluster) HTTP and HTTPS services can be accessed from applications inside the mesh. The [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task demonstrates how to allow the applications to send HTTP requests to external servers that require HTTPS. The [Configure an Egress Gateway](/docs/tasks/traffic-management/egress-gateway/) task describes how to configure Istio to direct egress traffic through a dedicated gateway service called _egress gateway_.
```

Comment:
allow the applications -> allow applications

Reply:
This paragraph may be somewhat redundant with the auto-generated See Also section, you may wish to delete it.

Reply:
@geeknoid I am OK to remove the Related tasks section, I wonder how the See Also section will be generated. What should I do to generate it? My thought was to present a sequence of related tasks, not as bullets, but as a sequence.

Reply:
The See Also section is auto-generated based on the keywords assigned to this and other pages. You can see it in the netlify preview, at the bottom of the page.

Reply:
Hmmm, Hugo didn't find any related pages to this one, so it no See Also section is produced for this page. Strange. I think our keywords may need more attention.